config/application.rb

```diff
@@ -33,5 +33,7 @@ class Application < Rails::Application
     config.active_storage.variant_processor = :mini_magick
 
     config.active_support.to_time_preserves_timezone = :zone
+
+    Faker::Config.locale = 'pt-BR'
```

Comment:
The faker is not used in production! You should define this line in `config/environments/development.rb`